README.md

```diff
@@ -1,111 +1,63 @@
-
-**SuperCollider** is a platform for audio synthesis and algorithmic composition, used by musicians, artists, and researchers working with sound. It is free and open source software available for Windows, macOS, and Linux.
-
-SuperCollider features three major components:
+**SuperCollider** is a platform for audio synthesis and algorithmic composition, used by musicians, artists, and researchers working with sound. It consists of:
 
 - scsynth, a real-time audio server, forms the core of the platform. It features 400+ unit generators ("UGens") for analysis, synthesis, and processing. Some of the audio techniques it supports include additive synthesis, subtractive, FM, granular, FFT, and physical modeling. You can write your own UGens in C++, and users have already contributed several hundred more to the sc3-plugins repository.
 - sclang, an interpreted programming language, controls scsynth via Open Sound Control. You can use sclang for algorithmic sequencing, connecting your app to external hardware including MIDI controllers, or writing GUIs and visual displays. sclang has a stock of user-contributed extensions called Quarks.
 - scide is an editor for sclang with an integrated help system.
 
-SuperCollider was developed by James McCartney and originally released in 1996. In 2002, he generously released it as free software under the GNU General Public License. It is now maintained and developed by an active and enthusiastic community.
+Install
+-------
 
-http://supercollider.github.io/
+macOS and Windows builds for stable releases are provided at our [downloads page][downloads page]. See the [macOS README](README_MACOS.md) and [Windows README](README_WINDOWS.md) for instructions on usage, and how to build SC yourself.
 
-This README is the first help for anyone new to SuperCollider. There are many
-good resources available for learning SuperCollider, with up to date links from
-the site listed above, including links to forums and mailing lists.
+To get the latest stable version, Linux users will need to build SuperCollider themselves. See the [Linux README](LINUX_README.md) for instructions.
 
+[downloads page]: https://supercollider.github.io/download
 
-Usage
+Learn
 -----
 
-SuperCollider consists of three separate components:
-
-1. scsynth or supernova - audio engine (the "server")
-2. sclang - programming language runtime interpreter including Qt graphical user interfaces
-3. IDE (integrated development environment) - an editor for writing code and running supercollider
-
-To begin using SuperCollider, you usually start up the IDE:
+The official SC docs can be viewed in the SuperCollider IDE's built-in help viewer. A copy is also hosted online at [doc.sccode.org](http://doc.sccode.org/).
```

Comment:
"help viewer" -> "documentation browser"? "A copy is also hosted online" -> "You can also view them online"? Also consider removing "SC" from "official SC docs", it's already clear from context.